### PR TITLE
Descriptions for Cohort fields

### PR DESCRIPTION
**Pull request by dtgreiner**
## Description

Add ability to edit description fields for cohorts.

## Type of change
- [x] Bug fix
- [x] New feature (adds functionality)
- [ ] Code clean-up / housekeeping
- [ ] Dependency update

## Checklist before requesting review
- [x] I have performed a self-review of my code
- [x] I have run the code that is being changed under ideal conditions, and it doesn't fail
- [x] My code includes comments and/or descriptive variable names to help other engineers understand the intent (or not applicable)
- [x] My code follows the style guidelines of this project (rubocop)
- [x] I have updated the documentation (or not applicable)
- [x] If it's not obvious how to test this change, I have provided testing instructions in this PR or the related issue

### CODE REVIEW

File: drivers/cas_ce_data/app/models/cas_ce_data/synthetic/event.rb [hunk 1]
```diff
@@ -43,7 +43,12 @@ def self.remove_orphans
     end
 
     def self.add_new
-      new_events = CasCeData::GrdaWarehouse::CasReferralEvent.where.not(id: self.select(:source_id))
+      # Replaced the following with a lateral join for performance, leaving temporarily for easy reversal
+      # new_events = CasCeData::GrdaWarehouse::CasReferralEvent.where.not(id: self.select(:source_id))
+      existing = where('"cas_referral_events"."id" = source_id').select(:source_id).limit(1)
+      new_events = CasCeData::GrdaWarehouse::CasReferralEvent.
+        joins("LEFT JOIN LATERAL (#{existing.to_sql}) findings ON true").
+        where('findings.source_id is NULL')
```
Comment (eanders) [inline drivers/cas_ce_data/app/models/cas_ce_data/synthetic/event.rb]: I think this file snuck in from somewhere else.  @dtgreiner was this intentional?

Comment (dtgreiner) [reply drivers/cas_ce_data/app/models/cas_ce_data/synthetic/event.rb]: @eanders including this code was not intentional. I'm guessing it snuck in when i ran into an issue during a branch swap.

**Comment by eanders** [comment]
https://github.com/greenriver/hmis-warehouse/pull/3711/commits/26dbede4bcbb9fbde6fe133286881ad85745abcd adds Tooltips using the description for the column headers.
Screenshot 2023-11-09 at 1 39 04 PM

**Review by eanders — APPROVED** [review]
Thanks @dtgreiner for the great work on this!

**Pull request merged**